### PR TITLE
Re-add use_l2pop attribute

### PR DESCRIPTION
**Pull request by vuntz**
Quoting @dirkmueller:

> l2pop with HA/vxlan just doesn't seem to work at all. Only enable what upstream does in devstack: when DVR is used.

This should replace https://github.com/crowbar/crowbar-openstack/pull/684

### CODE REVIEW

File: chef/data_bags/crowbar/migrate/neutron/105_add_back_use_l2pop.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+def upgrade(ta, td, a, d)
+  unless a.key? "use_l2pop"
+    # Only enable it if DVR is used
+    a["use_l2pop"] = a["use_dvr"]
```
Comment (aplanas) [inline chef/data_bags/crowbar/migrate/neutron/105_add_back_use_l2pop.rb]: why is not better to add this logic when we export the variable `use_l2pop` in the templates? if dvr are enabled or disabled after the barclamp, use_l2pop will not change with this approach

Comment (vuntz) [inline chef/data_bags/crowbar/migrate/neutron/105_add_back_use_l2pop.rb]: Because some users want to use l2pop without DVR.

Comment (aplanas) [inline chef/data_bags/crowbar/migrate/neutron/105_add_back_use_l2pop.rb]: Reading the quote from dirk looks like that this option was is not OK outside DVR.

Comment (vuntz) [inline chef/data_bags/crowbar/migrate/neutron/105_add_back_use_l2pop.rb]: It is okay, but it comes with some tradeoffs so people might not always want it.

**Pull request merged**